src/main.rs

```diff
@@ -229,23 +230,28 @@ fn main() {
 
     let authentication = Authentication::new();
     match &cli.command {
-        Commands::Login { token } => match authentication.verify_and_store_token(token) {
-            Ok(()) => {
-                info!("Login credentials have been saved.");
-                std::process::exit(0);
-            }
-
-            Err(e) => match e {
-                AuthenticationError::WrongCredentials => {
-                    error!("Token verification failed.");
-                    std::process::exit(1);
+        Commands::Login {} => {
+            print!("Enter your API Token: ");
```

Comment:
Could it be replaced by info!?

Reply:
No, because that will add a bunch of visual trash to the line that tasks to user